src/promptflow-index/setup.py

```diff
@@ -0,0 +1,68 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+import os
+import re
+from io import open
+from typing import Any, List, Match, cast
+
+from setuptools import find_namespace_packages, setup
```

Comment:
add test for this pacakge?

Reply:
This is a known gap in our product, and we have work item to address this. To meet the current deadline for build, I will come back to add tests once done with the rest of the work for index sdk migration.